### DISCUSSION TITLE
blog/how-to-hash-class-names-in-astro-using-astro-rename-integration/

### DISCUSSION THREAD

**Discussion by giscus** [LucJosin/lucasjosino.com#19] [Announcements]
# blog/how-to-hash-class-names-in-astro-using-astro-rename-integration/

Learn how to use the astro-rename integration to hash all CSS class names from your Astro project

https://www.lucasjosino.com/blog/how-to-hash-class-names-in-astro-using-astro-rename-integration/

**Comment by LucJosin** [comment]
*Hello* **World**!
